pkg/arm/resources.go

```diff
@@ -591,15 +591,17 @@ func Vmss(pc *api.PluginConfig, cs *api.OpenShiftManagedCluster, app *api.AgentP
 		Location: to.StringPtr(cs.Location),
 	}
 
+	// The additional data disk have different purposes on master
+	// and infra/compute nodes, but they are both the same size
+	vmss.VirtualMachineProfile.StorageProfile.DataDisks = &[]compute.VirtualMachineScaleSetDataDisk{
+		{
+			Lun:          to.Int32Ptr(0),
+			Caching:      compute.CachingTypesReadOnly,
+			CreateOption: compute.DiskCreateOptionTypesEmpty,
+			DiskSizeGB:   to.Int32Ptr(256),
```

Comment:
We might want to variablize the size `256` so that we can configure this more easily.  Or at least make a constant.

Reply:
Please can you push this up to be part of the `vmss := &compute.VirtualMachineScaleSet{` statement above?